stream.js

```diff
@@ -37,6 +37,10 @@ page.onError = function (err, trace) {
 	console.error('WARN: ' + err + formatTrace(trace[0]));
 };
 
+page.onResourceError = function (resourceError) {
+	console.error('WARN: Unable to load resource #' + resourceError.id + ' → ' + resourceError.url);
```

Comment:
Yep, makes sense – feel free to update it, or let me know what format you want the error message to take and I'll push a change.